<launch>
    <node pkg="path_planning" name="fake" type="fake.py" output="screen">
    </node>
    <node pkg="path_planning" name="path_planning" type="run_TPP.py" output="screen">
    </node>
</launch>
